src/mame/includes/tsconf.h

```diff
@@ -10,33 +10,32 @@
 
 #pragma once
 
-#include "spectrum.h"
-
 #include "machine/beta.h"
-
 #include "machine/glukrs.h"
 #include "machine/pckeybrd.h"
 #include "machine/spi_sdcard.h"
 #include "machine/tsconfdma.h"
-
+#include "spectrum.h"
```

Comment:
This header should come first, because we put most dependent module headers first to make it easier to catch headers not including everything they depend on.